playwright/helpers/sanityClient.ts

```diff
@@ -0,0 +1,13 @@
+import sanityClient from '@sanity/client'
+import {DEFAULT_DATASET, STUDIO_PROJECT_ID} from '.'
+
+export const testSanityClient = sanityClient({
+  projectId: 'ppsg7ml5',
+  dataset: 'test',
+  token: process.env.PLAYWRIGHT_SANITY_SESSION_TOKEN,
+  useCdn: false,
+})
+
+export async function deleteDocument(payload) {
+  return testSanityClient.delete(payload)
+}
```

Comment:
I feel like we can just as well use `testSanityClient.delete()` directly to get more control of the operation (second argument, for instance), or have this function do something more specific, eg `function deleteDocumentsForRun(typeName: string)`, which could generate a GROQ query with the correct type name and run ID. Thoughts?